backend/src/api/disk-cache.ts

```diff
@@ -5,11 +5,13 @@ import memPool from './mempool';
 import blocks from './blocks';
 import logger from '../logger';
 import config from '../config';
+import { TransactionExtended } from '../mempool.interfaces';
 
 class DiskCache {
   private static FILE_NAME = config.MEMPOOL.CACHE_DIR + 'cache.json';
   private static FILE_NAMES = config.MEMPOOL.CACHE_DIR + 'cache{number}.json';
```

Comment:
Probably a good idea to add the trailing slash here, instead of in the configuration
```suggestion
  private static FILE_NAME = config.MEMPOOL.CACHE_DIR + '/cache.json';
  private static FILE_NAMES = config.MEMPOOL.CACHE_DIR + '/cache{number}.json';
```